### PR TITLE
Pull Request for Issue #87: Integrate generate_bed.py  into PanelPal build

### PR DESCRIPTION
**Pull request by ASendellPrice**
## Overview
This pull request integrates generate_bed.py into PanelPal build. Generation of bed files can now be run using the following command:
```
PanelPal generate-bed --panel_id R207 --panel_version 4 --genome_build GRCh38
```
This resolves issue #87 

## Main changes
- Updated main() function of generate_bed.py allowing arguments to be passed directly from PanelPal
- Updated README.md to include PanelPal method for running generate_bed.py
- Improved usage information message produced by PanelPal:
image

## Unit testing
- All unit tests passed successfully
image

### CODE REVIEW

File: PanelPal/main.py
Comment (PatrickWeller) [inline PanelPal/main.py:1]: You've lost your doc string! Add it back please :)

File: PanelPal/main.py [hunk 1]
```diff
@@ -1,43 +1,37 @@
...
+    """
+    print(help_message)
+
+
 def main():
-    """Main function which gathers arguments and passes them to the relevant PanelPal command"""
+    """Main function which gathers arguments and passes them to the relevant PanelPal command."""
     parser = argparse.ArgumentParser(
-        description="PanelPal: A toolkit for panelapp queries"
+        description="PanelPal: A toolkit for querying and generating genomic panel BED files.",
```
Comment (PatrickWeller) [inline PanelPal/main.py:33]: Not just for querying and generating bed files

File: PanelPal/main.py [hunk 2]
```diff
@@ -1,43 +1,37 @@
...
+        epilog="For more details, visit https://yourprojecturl.com"
```
Comment (PatrickWeller) [inline PanelPal/main.py:34]: Put actual repository link

**Review by PatrickWeller — CHANGES_REQUESTED** [review]
Hi Ash, just a couple of changes for you, thanks

File: PanelPal/main.py
Comment (ASendellPrice) [reply PanelPal/main.py:1]: Addressed in commit [23629c9](https://github.com/PatrickWeller/PanelPal/pull/88/commits/23629c9d06f57d449c9e14bb0725b388dffb8d96)

File: PanelPal/main.py (hunk 1 above)
Comment (ASendellPrice) [reply PanelPal/main.py:33]: Addressed in commit [23629c9](https://github.com/PatrickWeller/PanelPal/pull/88/commits/23629c9d06f57d449c9e14bb0725b388dffb8d96)

File: PanelPal/main.py (hunk 2 above)
Comment (ASendellPrice) [reply PanelPal/main.py:34]: Addressed in commit [23629c9](https://github.com/PatrickWeller/PanelPal/pull/88/commits/23629c9d06f57d449c9e14bb0725b388dffb8d96)

**Review by PatrickWeller — APPROVED** [review]
Happy with changes, I will merge, thanks Ash

**Pull request merged**